### PR TITLE
github action to copy Nevada hospital data from a Google Sheet to this repo

### PR DESCRIPTION
**Pull request by TomGoBravo**
Discussed at

https://trello.com/c/oGNg8jGK/17-update-icu-hospital-data-from-daily-nevada-emails
and
https://covidactnow.slack.com/archives/G013M5H6Z8R/p1588873790004400

Tested the script by running manually. github action is untested.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,3 +5,4 @@ pandas==1.0.3
 pytz==2019.3
 requests==2.23.0
 urllib3==1.25.8
+python-dateutil==2.8.1
```
Comment (ghop02) [inline requirements.txt:8]: I think you might have to add pydantic as well!

File: scripts/update_nha_hospitalization_county.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    class Config:
+        arbitrary_types_allowed = True  # For PathLike
+
+    def update(self):
+        _logger.info(f'Copying {self.source_url} to {self.destination_path}')
+        response = requests.get(self.source_url)
+        patched_rows = []
+
+        for row in csv.reader(response.iter_lines(decode_unicode=True)):
```
Comment (ghop02) [inline scripts/update_nha_hospitalization_county.py:28]: We'd need to change the code in covid-data-model just a bit, but do you think we should skip the first row? It contains the not headers and generally makes this a bit more difficult to work with

File: requirements.txt (hunk 1 above)
Comment (TomGoBravo) [reply requirements.txt:8]: Good catch! I'm using one pyenv for all repos but perhaps that is asking for trouble.

Comment (ghop02) [reply requirements.txt:8]: I've landed on the setup of using pyenv-virtualenv to make it quick and easy to work with multiple venvs: https://github.com/covid-projections/covid-data-model/blob/master/SETUP.md

would recommend!

File: scripts/update_nha_hospitalization_county.py (hunk 2 above)
Comment (TomGoBravo) [reply scripts/update_nha_hospitalization_county.py]: Sounds good. Done.

**Comment by TomGoBravo** [comment]
This change has a bunch of commits going to master. "CoVid Act Now - Engineering Onboarding" talks about rebasing (which I've done) and merging to `develop`. Is that relevant to this PR?

**Comment by ghop02** [comment]
ah no develop here - that's just in the webapp.  the squash merge should do it!

**Pull request merged**